pkg/generators/models/testdata/cases/allof_enum/generated/model_any_thing.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 // AnyThing is an enum.
-type AnyThing Food
+type AnyThing string
```

Comment:
this was actually kind of a bug, `Anything` shouldn't have been an alias to just one of the enums it was composed from.